subgraphs/abracadabra/src/common/utils/arrays.ts

```diff
@@ -1,3 +1,33 @@
+export function removeFromArrayAtIndex<T>(x: T[], index: i32): T[] {
```

Comment:
This is nice! Would you be able to add these to the `_reference_` subgraph?

Reply:
Sure, I will create another PR for this!

Reply:
https://github.com/messari/subgraphs/pull/493